pkg/rpc/client/client_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"errors"
 	"testing"
 
+	ftypes "github.com/aquasecurity/fanal/types"
+
```

Comment:
nit: I think we don't have to change the import order like this.

Reply:
It's my IDE that thinks this is the right order. I can revert back. I don't think it's a big deal though.